hub/service/sweep_service.cc

```diff
@@ -286,6 +287,11 @@ bool SweepService::doTick() {
         hubInputs.clear();
         requiredOutput = 0;
         withdrawals.clear();
+      } else {
+        std::vector<db::TransferInput> minimalVecOfInputs;
+        getVecOfMinSizeWithSumNotLessThan(missing, hubInputs,
+                                          minimalVecOfInputs);
+        hubInputs.swap(minimalVecOfInputs);
```

Comment:
You also need to amend `hubInputTotal` as this could vary (required for `remainder` calculation)

Reply:
Still missing the change here

Reply:
what do you mean by ` amend hubInputTotal` ?

Reply:
ha

Reply:
`hubInputTotal` needs to be adjusted with the sum of `minimalVecOfInputs.balance`.

This is relevant for the `remainder` on l310